components/resources/wallet_strings.grdp

```diff
@@ -413,7 +413,7 @@
   <message name="IDS_BRAVE_WALLET_TRANSACTION_EXPLORER_MISSING" desc="Alert for when no block explorer is available">Block explorer URL is not available.</message>
   <message name="IDS_BRAVE_WALLET_TRANSACTION_EXPLORER" desc="View on block explorer button">View on block explorer</message>
   <message name="IDS_BRAVE_WALLET_TRANSACTION_COPY_HASH" desc="Copy transaction hash button">Copy transaction hash</message>
-  <message name="IDS_BRAVE_WALLET_TRANSACTION_SPEEDUP" desc="Speedup transaction button">Speedup transaction</message>
+  <message name="IDS_BRAVE_WALLET_TRANSACTION_SPEEDUP" desc="Speedup transaction button">Speedup</message>
```

Comment:
Nit: `Speed up` (as in a verb vs a noun ), assuming this is meant as a request to speed up.

Reply:
Updated to `Speed up`